black.html

```diff
@@ -0,0 +1,94 @@
+
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <script src="bootstrap/js/bootstrap.min.js"></script>
+
+  </head>
+  <body>
+    <div class="container">
+
+      <!-- Static navbar -->
+      <nav class="navbar navbar-default">
+        <div class="container-fluid">
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+              <span class="sr-only">Toggle navigation</span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+            <!-- Unused brand for page -->
+            <!-- <a class="navbar-brand" href="#">Magic .Cards</a> -->
+          </div>
+          <!-- Navigation bar with mana symbols, text versions are commented out -->
+          <div id="navbar" class="navbar-collapse collapse">
+            <ul class="nav navbar-nav">
+              <li><a href="index.html"><img src="home-symbol.jpg" alt="Home" class="size"></a></li>
+              <li><a href="white.html"><img src="mana-symbols/white-mana.png" alt="White Mana" class="img-circle size"></a></li>
+              <li><a href="blue.html"><img src="mana-symbols/blue-mana.png" alt="Blue Mana" class="img-circle size"></a></li>
+              <li><a href="black.html"><img src="mana-symbols/black-mana.png" alt="Black Mana" class="img-circle size"></a></li>
+              <li><a href="red.html"><img src="mana-symbols/red-mana.png" alt="Red Mana" class="img-circle size"></a></li>
+              <li><a href="green.html"><img src="mana-symbols/green-mana.png" alt="Green Mana" class="img-circle size"></a></li>
+              <li><a href="multicolor.html"><img src="mana-symbols/multicolor-mana.png" alt="Multicolor Mana" class="img-circle size"></a></li>
+              <li><a href="colorless.html"><img src="mana-symbols/colorless-mana.png" alt="Colorless Mana" class="img-circle size"></a></li>
+              <li><a href="https://magic.wizards.com/" target="_blank"><img src="mtg-symbol.png" alt="MTG symbol" class="mtgSize"></a></li>
+              </li>
+            </ul>
+          </div><!--/.nav-collapse -->
+        </div><!--/.container-fluid -->
+      </nav>
+
+      <div class="container">
+    <div id="myCarousel" class="carousel slide" data-ride="carousel">
+  <!-- Indicators -->
+  <ol class="carousel-indicators">
+    <li data-target="#myCarousel" data-slide-to="0"></li>
+    <li data-target="#myCarousel" data-slide-to="1"></li>
+    <li data-target="#myCarousel" data-slide-to="2"></li>
+  </ol>
+
+  <!-- Wrapper for slides -->
+  <div class="carousel-inner">
+    <div class="item active">
```

Comment:
I'd like to see a little more content describing what each card is and what it does/what's interesting about it.